pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -685,17 +685,19 @@ func startConsumer(t *testing.T, svc *Service, done chan bool) {
 
 	go func() {
 		for e := range actionCh {
+			// assigned to var as lint fails with : Using a reference for the variable on range scope (scopelint)
```

Comment:
Linter fails because each iteration of the loop overwrites e.

When you use a pointer to e, you need to be careful that each iteration actually dereferences the pointer rather than using the pointer as a value.